### PR TITLE
build(aio): update project config for @angular/cli v1.0.0

### PR DESCRIPTION
**Pull request by gkalpak**
This is a follow-up to 487a0e1. The changes are based on [this wiki entry][1].
(I also removed the dependency on `gulp`, which is not used any more.)

[1]: https://github.com/angular/angular-cli/wiki/stories-1.0-update.

### CODE REVIEW

File: aio/src/tsconfig.spec.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      "jasmine",
+      "node"
+    ]
+  },
+  "files": [
+    "test.ts"
+  ],
+  "include": [
+    "testing/**/*.ts",
+    "**/*.spec.ts",
```
Comment (petebacondarwin) [inline aio/src/tsconfig.spec.json]: doesn't the `test.ts` import all these files?

Comment (petebacondarwin) [inline aio/src/tsconfig.spec.json]: but I see this is what it suggests in the wiki

**Review by petebacondarwin — APPROVED** [review]

File: aio/src/tsconfig.spec.json (hunk 1 above)
Comment (gkalpak) [inline aio/src/tsconfig.spec.json]: That how the cli does it (e.g. on a newly generated app). I think test.ts loads the modules (i.e. `require`s them), but for TS purposes, you need to include them in tsconfig.

File: aio/.angular-cli.json [hunk 2]
```diff
@@ -55,19 +57,8 @@
...
-    },
-    "spec": {
-      "class": false,
-      "component": true,
-      "directive": true,
-      "module": false,
-      "pipe": true,
-      "service": true
+    "component": {
+      "inlineStype": true
```
Comment (mhheise) [inline aio/.angular-cli.json]: Typo? `inlineStype` -> `inlineStyle`?

Comment (petebacondarwin) [inline aio/.angular-cli.json]: good spot!

**Pull request merged**